### PR TITLE
Updating docs per Andrey.

### PR DESCRIPTION
**Pull request by rivkahstandig3636**
There are some areas that I have unresolved questions about. I'll comment on the lines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,42 @@ new RollbarConfig
...
+</dt>
+<dd>Function called before sending payload to Rollbar. Return `true` to stop the error from being sent to Rollbar.
+</dd>
+<dt>Truncate
+</dt>
+<dd>Truncates the payload before sending it to Rollbar.
+</dd>
+<dt>Server
+</dt>
+<dd>The server name, as a string.
```
Comment (rivkahstandig3636) [inline README.md]: I'm unclear on if this is accurate or not.

File: README.md [hunk 2]
```diff
@@ -70,6 +70,42 @@ new RollbarConfig
...
+
+and this person will be attached to all future Rollbar calls.
+</dd>
+<dt>MaxReportsPerMinute
+</dt>
+<dd>The maximum reports sent to Rollbar per minute, as an integer.
+</dd>
+<dt>ReportingQueueDepth
+</dt>
+<dd>The reporting queue depth, as an integer.
```
Comment (rivkahstandig3636) [inline README.md]: I'd like more information about what exactly this means.

**Comment by rivkahstandig3636** [comment]
Additionally, we need to implement either ScrubFields or some other scrub functionality and add this to the docs. @akornich @rokob Please review this for accuracy.

File: README.md [hunk 3]
```diff
@@ -70,6 +70,42 @@ new RollbarConfig
 </dt>
 <dd>A string URI which will be used as the WebClient proxy by passing to the WebProxy constructor.
 </dd>
+<dt>CheckIgnore
+</dt>
+<dd>Function called before sending payload to Rollbar. Return `true` to stop the error from being sent to Rollbar.
```
Comment (rivkahstandig3636) [inline README.md]: can i get a code example to add here?

File: README.md [hunk 4]
```diff
@@ -70,6 +70,42 @@ new RollbarConfig
...
+</dd>
+<dt>Truncate
+</dt>
+<dd>Truncates the payload before sending it to Rollbar.
```
Comment (rivkahstandig3636) [inline README.md]: a code example here as well please!

File: README.md (hunk 2 above)
Comment (akornich) [reply README.md]: The way new notifier works is:
Multiple instances of RollbarLogger can be created. Each instance does not really send payloads when its reporting methods are called. It just creates payload object and drops it into a singleton dictionary of queues that are indexed by unique access_token values. The RollbarQueue controller is in charge of actually sending/retrying/etc. of the payloads across all the queues while obeying reporting rate limits. The queues can be assigned a queue depth to limit reports bursts when connectivity to the Rollbar API server is too poor.  
More details are in the attached file (please, see the "Conceptual Design of the Notifier" section)
[Rollbar.NET Notifier Documentation.docx](https://github.com/rollbar/Rollbar.NET/files/1617780/Rollbar.NET.Notifier.Documentation.docx)

File: README.md (hunk 4 above)
Comment (akornich) [reply README.md]: any user-defined  .NET function delegate implementing following signature 
bool FunctionName(Payload payload)
can be assigned to this property.

File: README.md (hunk 3 above)
Comment (akornich) [reply README.md]: any user-defined  .NET function delegate implementing following signature 
bool FunctionName(Payload payload)
can be assigned to this property.

File: README.md (hunk 4 above)
Comment (akornich) [reply README.md]: similar description goes for Transform action property...

File: README.md (hunk 1 above)
Comment (akornich) [reply README.md]: I am not sure where this one comes from. We have complex DTO type called Server that has following properties:
        public static class ReservedProperties
        {
            /// <summary>
            /// The host
            /// </summary>
            public const string Host = "host";
            /// <summary>
            /// The root
            /// </summary>
            public const string Root = "root";
            /// <summary>
            /// The branch
            /// </summary>
            public const string Branch = "branch";
            /// <summary>
            /// The code version
            /// </summary>
            public const string CodeVersion = "code_version";
        }

**Comment by rivkahstandig3636** [comment]
@akornich Should be good to go, please review one last time :)

**Comment by akornich** [comment]
I'll merge it as is and then make the requested corrections.

**Pull request merged**